### PR TITLE
Enhancement/1170/make pipeline 217 compatible

### PR DESCRIPTION
**Pull request by richp405**
##### SUMMARY
Changes to 1.10 ignore and sanity test include for Ansible 2.17 compatibility

Fixes 1170

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
tests/sanity needed ignore-2.17 file
Also tested on test_zos_data_set_func.py, which exemplified the need for a change to import.

### CODE REVIEW

**Comment by richp405** [comment]
image

File: tests/functional/modules/test_zos_data_set_func.py [hunk 1]
```diff
@@ -18,7 +18,8 @@
 import pytest
 import time
 import subprocess
-from pipes import quote
+# from pipes import quote
```
Comment (fernandofloresg) [inline tests/functional/modules/test_zos_data_set_func.py:21]: We can just remove this

**Comment by fernandofloresg** [comment]
Got a question, why is this tested only in one test case? I searched for quote() function and is used in multiple tests. I think we should at least test those right?

**Review by fernandofloresg — APPROVED** [review]
Looks good to me

**Pull request merged**